bootstrap/src/lib/cache/local.ts

```diff
@@ -1,45 +1,44 @@
-const LRU = require('lru-cache')
-const { parseBool } = require('../util')
+import LRU from 'lru-cache'
+import { parseBool } from '../util'
 
 // Options
 const DEFAULT_CACHE_MAX_ITEMS = 500
 const DEFAULT_CACHE_MAX_AGE = 1000 * 30 // Maximum age in ms
 const DEFAULT_CACHE_UPDATE_AGE_ON_GET = false
+const DEFAULT_CACHE_TIMEOUT = 500 // Timeout in ms
 
 const env = process.env
-const defaultOptions = () => ({
+export const defaultOptions = () => ({
   max: Number(env.CACHE_MAX_ITEMS) || DEFAULT_CACHE_MAX_ITEMS,
   maxAge: Number(env.CACHE_MAX_AGE) || DEFAULT_CACHE_MAX_AGE,
   updateAgeOnGet: parseBool(env.CACHE_UPDATE_AGE_ON_GET) || DEFAULT_CACHE_UPDATE_AGE_ON_GET,
+  timeout: DEFAULT_CACHE_TIMEOUT,
```

Comment:
Local cache impl does not have (or need) this `timeout` that our Redis impl has.

```suggestion
```